packages/flutter/lib/src/widgets/inherited_model.dart

```diff
@@ -6,53 +6,61 @@ import 'dart:collection';
 
 import 'framework.dart';
 
-/// An [InheritedWidget] that's intended to be used as the base class for
-/// models whose dependents may only depend on one part or "aspect" of the
-/// overall model.
+/// An [InheritedWidget] that's intended to be used as the base class for models
+/// whose dependents may only depend on one part or "aspect" of the overall
+/// model.
 ///
 /// An inherited widget's dependents are unconditionally rebuilt when the
-/// inherited widget changes per [InheritedWidget.updateShouldNotify].
-/// This widget is similar except that dependents aren't rebuilt
-/// unconditionally.
+/// inherited widget changes per [InheritedWidget.updateShouldNotify]. This
+/// widget is similar except that dependents aren't rebuilt unconditionally.
 ///
-/// Widgets that depend on an [InheritedModel] qualify their dependence
-/// with a value that indicates what "aspect" of the model they depend
-/// on. When the model is rebuilt, dependents will also be rebuilt, but
-/// only if there was a change in the model that corresponds to the aspect
-/// they provided.
+/// Widgets that depend on an [InheritedModel] qualify their dependence with a
+/// value that indicates what "aspect" of the model they depend on. When the
+/// model is rebuilt, dependents will also be rebuilt, but only if there was a
+/// change in the model that corresponds to the aspect they provided.
 ///
 /// The type parameter `T` is the type of the model aspect objects.
 ///
 /// {@youtube 560 315 https://www.youtube.com/watch?v=ml5uefGgkaA}
 ///
 /// Widgets create a dependency on an [InheritedModel] with a static method:
-/// [InheritedModel.inheritFrom]. This method's `context` parameter
-/// defines the subtree that will be rebuilt when the model changes.
-/// Typically the `inheritFrom` method is called from a model-specific
-/// static `of` method. For example:
+/// [InheritedModel.inheritFrom]. This method's `context` parameter defines the
+/// subtree that will be rebuilt when the model changes. Typically the
+/// `inheritFrom` method is called from a model-specific static `maybeOf` or
+/// `of` methods, a convention that is present in many Flutter framework classes
+/// which look things up. For example:
 ///
 /// ```dart
 /// class MyModel extends InheritedModel<String> {
 ///   const MyModel({super.key, required super.child});
+///
 ///   // ...
-///   static MyModel? of(BuildContext context, String aspect) {
+///   static MyModel? maybeOf(BuildContext context, String? aspect) {
```

Comment:
I would probably change the example to 

```suggestion
///   static MyModel? maybeOf(BuildContext context, [String? aspect]) {
```

since calling this with `MyModel.maybeOf(context, null)` looks strange and `MyModel.maybeOf(context)` looks a lot cleaner if you actually want to support null.